.github/workflows/lint.yml

```
@@ -0,0 +1,50 @@
+name: Lint
+
+on: pull_request
+
+jobs:
+  lint-commits:
+    name: Lint commits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/cache@v2
```

Update to v3 to use Node16 instead of Node12, https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/

```suggestion
      - uses: actions/cache@v3
```

---

Oops. I thought I changed all these. Thanks for catching this one.